contribs/vsp/src/test/java/org/matsim/contrib/freight/analysis/MyShipmentTrackerEventHandler.java

```diff
@@ -38,18 +37,18 @@ private void init(){
     }
 
     @Override
-    public void handleEvent(ShipmentDeliveredEvent event) {
+    public void handleEvent(FreightShipmentDeliveryEndEvent event) {
         shipmentTracking.trackDeliveryEvent(event);
     }
 
     @Override
-    public void handleEvent(ShipmentPickedUpEvent event) {
+    public void handleEvent(FreightShipmentPickupEndEvent event) {
         shipmentTracking.trackPickedUpEvent(event);
     }
 
     @Override
     public void reset(int iteration) {
-        ShipmentPickedUpEventHandler.super.reset(iteration);
+        FreightShipmentPickupEventHandler.super.reset(iteration);
```

Comment:
This looks weird what does this do? I don't know what this notation does. Just think about whether this does what you want.